### PR TITLE
Replace makefile.shade with a PowerShell script

### PR DESCRIPTION
**Pull request by natemcmaster**
Replace the custom build steps for building a VSIX with a powershell script.

cref aspnet/Korebuild#174 and https://github.com/aspnet/KoreBuild/pull/180

### CODE REVIEW

File: build/repo.targets [hunk 1]
```diff
@@ -1,8 +1,12 @@
 <Project>
+  <!--
+    TODO include VSIX build in default lifecycle builds
```
Comment (natemcmaster) [inline build/repo.targets]: I chose not to as we only have one CI agent with compatible build tools installed.

Comment (pranavkm) [inline build/repo.targets]: The plan is to have VS 2017 installed on all our build agents. We're avoiding the hassle of updating them later by installing a RC build now.

Comment (pranavkm) [inline build/repo.targets]: See https://github.com/aspnet/Coherence-Signed/issues/409

Comment (natemcmaster) [inline build/repo.targets]: Cool. When we finish that, we can come back and take care of this TODO.

**Pull request merged**